update_image.pl

```diff
@@ -375,24 +387,27 @@ sub processConvergedSections {
     {
         if(-e $ocmbfw_original_filename)
         {
-            #Verify image validity
-            run_command("$hb_image_dir/pkgOcmbFw.pl --verify --packagedBin $ocmbfw_original_filename");
-            $sections{OCMBFW}{in}    = "$ocmbfw_original_filename";
+            #Add header to delivered image
+            my $date = `date`;
+            run_command("$hb_image_dir/pkgOcmbFw.pl --unpackagedBin $ocmbfw_original_filename --packagedBin $ocmbfw_original_filename.header --timestamp \"$date\" --vendorVersion \"$ocmbfw_version\" --vendorUrl \"$ocmbfw_url\"");
```

Comment:
so this cmd does add a sha512 header to the image?  and it (obviously) includes the vendorVersion and vendorUrl  to that header?